notebooks/community/ml_ops/stage1/get_started_with_visionapi_and_vertex_datasets.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>you already imported datetime when you did the TIMESTAMP</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/683/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>